docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
+parent: 8. Top cover and doors panels
+grand_parent: Vz235 - Mellow Kit
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
+---
+
+# 8.1 Top Cover Assembly
+
+## Overview
+
+![Overview](../../assets/images/manual/vz235_printed/top_cover/top_cover_overview.png)
+
+## BOM
+
+| Material                    | Quantity | Notes                                                |
+| --------------------------- | -------- | ---------------------------------------------------- |
+| M3 x 8 BHCS                 | 8        |                                                      |
+| M3 Nut                      | 8        |                                                      |
+| M4 x 10 SHCS                | 8        |                                                      |
+| M4 hammer t-nut             | 8        |                                                      |
+| 2 x 8mm Magnet              | 8        |                                                      |
+| 395\*405\*3mm acrylic panel | 1        | Refer to the .step file for placement of the cutouts |
```

Comment:
One can also refer to the web Fusion360 model. The CNC and Printed versions are both linked at the top of the README.MD here: https://github.com/VzBoT3D/VzBoT-Vz235

Reply:
I can refer to that. I had just left the wording the same as what was already there.

Reply:
I have decided to remove that note as this is a kit after all and its supplied fully manufactured.